MarkdownApp/Shared/ViewModels/Interfaces/ILoginViewModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿using MarkdownApp.Shared.Models;
+namespace MarkdownApp.Shared.ViewModels.Interfaces
+{
+    public interface ILoginViewModel
```

Comment:
why do u need this interface?

Reply:
no, the idea was to create a repository or service rather than "viewModels" to call the controller, but end it up having a useless interfaces and classes.